ner_processing/master_mapping.py

```diff
@@ -0,0 +1,180 @@
+"""
+This file specifes the CAN and data ID mappings. There are three levels of IDs specified:
+    - External Message ID (actual CAN message id)
+    - Internal Message ID (internal id for messages to allow easier changing of external ids)
+    - Data ID (id for individual data values contained in the messages)
+"""
+
+from ner_processing.decode_data import *
+
+# Mapping from external to internal message IDs
+MESSAGE_IDS = {
```

Comment:
Is this a way of abstracting the CAN messages into easy-to-remember data structures, or will this doc have to be used as a key while developing processing code? Also what would be the method for adding/deleting CAN messages that we want to process?